tomviz/ModuleManager.cxx

```diff
@@ -355,6 +370,16 @@ bool ModuleManager::deserialize(const pugi::xml_node& ns, const QDir& stateDir)
       viewSummary.append_attribute("name").set_value(
           QString("View%1").arg(++numViews).toStdString().c_str());
     }
+    if (agproxyNode)
```

Comment:
As these are scoped, and local meaning is pretty obvious, if it isn't used simply proxyNode would work fine too (and be shorter). Either way, I think this should be cleaned up and the variable names between reading and writing kept consistent.